l10n/bundle.l10n.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This json file is generated by following command automatically:
npx @vscode/l10n-dev export -o ./l10n ./src